src/node/handler/ImportHandler.js

```diff
@@ -82,6 +82,12 @@ async function doImport(req, res, padId)
         if (err) {
           console.warn("Uploading Error: " + err.stack);
         }
+
+        // I hate doing indexOf here but I can't see anything to use...
+        if (err.stack.indexOf("maxFileSize") !== -1) {
+          reject("maxFileSize");
+        }
```

Comment:
Doesn't exist a way of properly bubbling the error?

The server logs are not so good looking when an import is rejected when a file is too big:

```
When the 
[2020-04-21 22:02:53.499] [WARN] console - Uploading Error: Error: maxFileSize exceeded, received 11353 bytes of file data
    at Stream.<anonymous> (<BASE>/src/node_modules/formidable/lib/incoming_form.js:222:19)
    at Stream.emit (events.js:210:5)
    at Stream.EventEmitter.emit (domain.js:475:20)
    at MultipartParser.parser.onPartData (<BASE>/src/node_modules/formidable/lib/incoming_form.js:387:14)
    at callback (<BASE>/src/node_modules/formidable/lib/multipart_parser.js:102:31)
    at dataCallback (<BASE>/src/node_modules/formidable/lib/multipart_parser.js:115:11)
    at MultipartParser.write (<BASE>/src/node_modules/formidable/lib/multipart_parser.js:237:15)
    at IncomingForm.write (<BASE>/src/node_modules/formidable/lib/incoming_form.js:159:34)
    at IncomingMessage.<anonymous> (<BASE>/src/node_modules/formidable/lib/incoming_form.js:125:12)
    at IncomingMessage.emit (events.js:210:5)
```

Reply:
No, this error is coming from formidable.

Reply:
I see. But a way to catch it must exist.

Let's skim on it for now (== forever). Closing.